README.md

```diff
@@ -70,82 +116,135 @@ Docker is needed to run the project in a containerized environment. Follow the a
 Ensure `docker-compose` is installed. Refer to the [Docker Compose installation guide](https://docs.docker.com/compose/install/).
 
 ### Setting Up Environment Variables
-Create or update the `.env` files in the `fastapi_backend` and `nextjs-frontend` directories with the following variables:
 
 **Backend (`fastapi_backend/.env`):**
-
-For local setup:
-```
-DATABASE_URL=postgresql+asyncpg://postgres:password@localhost:5432/mydatabase
-```
-For Docker setup, replace localhost with db:
-```
-DATABASE_URL=postgresql+asyncpg://postgres:password@db:5432/mydatabase
-```
-Other environment variables remain unchanged:
-
-```env
-ACCESS_SECRET_KEY=<your-secret-key>
-RESET_PASSWORD_SECRET_KEY=<your-secret-key>
-VERIFICATION_SECRET_KEY=<your-secret-key>
-OPENAPI_OUTPUT_FILE=../nextjs-frontend/openapi.json
-
-MAIL_USERNAME=<your-username>
-MAIL_PASSWORD=<your-password>
-MAIL_FROM=<your-from>
-MAIL_SERVER=<your-server>
+Copy the `.env.example` files to `.env` and update the variables with your own values.
```

Comment:
You have to explain what needs to be filled. Like: 
- Change DATABASE_URL if you're planning to use Docker to run the local servers
- Same for TEST_DATABASE_URL
- Generate SECRET_KEYs and save them (suggest how to generate, maybe with a python command like `python3 -c "import secrets; print(secrets.token_hex(32))"`)
- Set a SMTP mail server if you need

Reply:
Also, why don't we have a local test-only SMTP server?

Reply:
1. Sure!
2. Makes sense, I'll add a local server.

Reply:
Please, check if all is good now, the only things that the user needs to generate and change are the secrect keys.
I left a comment mentioning it. And there's an explanation in the .env file about the docker configuration.